pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = [ "setuptools", "setuptools_scm" ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ts_m2"
+description = """
+This package is the commandable SAL component (CSC) and provides a simple
+emulator for the M2 control system."""
+license = { text = "GPL" }
+classifiers = [ "Programming Language :: Python :: 3" ]
+urls = {documentation = "https://jira.lsstcorp.org/secure/Dashboard.jspa", repository = "https://github.com/lsst-ts/ts_m2" }
+dynamic = [ "version" ]
+
+[tool.setuptools.dynamic]
+version = { attr = "setuptools_scm.get_version" }
+
+[tool.setuptools.packages.find]
+where = [ "python" ]
+
+[project.scripts]
+run_mtm2 = "lsst.ts.m2.csc:run_mtm2"
+
+[tool.setuptools_scm]
+write_to = "python/lsst/ts/m2/version.py"
+write_to_template = """
+# Generated by setuptools_scm
+__all__ = ["__version__"]
+__version__ = "{version}"
+"""
+
+[tool.pytest.ini_options]
+addopts = "--black --flake8 --ignore-glob=*/version.py"
+flake8-ignore = ["E133", "E203", "E226", "E228", "N802", "N803", "N806", "N812", "N813", "N815", "N816", "W503", "doc/conf.py ALL", "__init__.py ALL", "version.py ALL", "tests/.tests ALL"]
```

Comment:
Note that in the latest version of the template in the confluence page, this line was updated as well to be much shorter. I'll leave it up to you to change it or not. Like this it is fine but it contains some superfluous entries that may be removed if you want.

Reply:
Updated in the commit: [835faac](https://github.com/lsst-ts/ts_m2/pull/25/commits/835faacd505ba3eb57001036a7757269e1fd4219). Thanks!